Xargo.toml

```diff
@@ -0,0 +1,5 @@
+[target.x86_64-uwp-windows-msvc.dependencies]
+std = { features = ["panic-unwind"] }
```

Comment:
Is this necessary? Unlike the ARM target (see comment below), this appears to default to keep the `PanicStrategy::Unwind` default: https://github.com/rust-lang/rust/blob/000d90b11f7be70ffb7812680f7abc6deb52ec88/src/librustc_target/spec/x86_64_uwp_windows_msvc.rs

Reply:
Xargo doesn't build libstd successfully unless one of panic-unwind or panic-abort is specified.